src/contexts/AuthContextProvider.jsx

```diff
@@ -16,6 +16,7 @@ const AuthContextProvider = ({ children }) => {
 	const [user, setUser] = useState(undefined);
 
 	useEffect(() => {
+		if (user) return;
```

Comment:
Denk dat deze fout is. Je gaat nu niks doen als de user is ingelogd en als er geen user is ga je data fetchen